text/2020-03-11-versioned-kv.md

```diff
@@ -77,13 +77,19 @@ VerKV is a kv-like interface similar to the RawKV interface. It hides version
 information from upper-layer services. Its data model is multi-versioned KV
 between RawKV and TxnKV.
 
-#### Data model
+#### Data model and Consistency
 
 The data model is like `key:version -> value`. The version uses the current
 timestamp by default, or can be specified by the user (such as the global `ts`
 obtained from the PD TSO). And the implementation is subject to further change
 if we use the user TS feature of RocksDB.
 
+For consistency, strong and weak consistency would be support. This is
```

Comment:
Much better! could you specify what strong consistency you refer to? Single key level linearizability or snapshot isolation?

Reply:
single key level linearlizability

Reply:
Thank you for the clarification!